util/regen.sh

```diff
@@ -5,6 +5,6 @@ qmk generate-rgb-breathe-table -o quantum/rgblight/rgblight_breathe_table.h
 qmk generate-keycodes --version latest -o quantum/keycodes.h
 qmk generate-keycodes-tests --version latest -o tests/test_common/keycode_table.cpp
 
-for lang in $(find data/constants/keycodes/extras/ -type f -printf "%f\n"  | sed "s/keycodes_\(.*\)_[0-9].*/\1/"); do
+for lang in $(find data/constants/keycodes/extras -type f -exec basename {} \; | sed "s/keycodes_\(.*\)_[0-9].*/\1/"); do
```

Comment:
```suggestion
for lang in $(find data/constants/keycodes/extras -type f -exec basename '{}' \; | sed "s/keycodes_\(.*\)_[0-9].*/\1/"); do
```

...someone's going to use it with spaces in their folder hierarchy at some point.